src/psyclone/transformations.py

```diff
@@ -1794,3 +2127,85 @@ def apply(self, nodes):
                              index=node_position)
 
         return schedule, keep
+
+
+class ACCDataTrans(Transformation):
+    '''
+    Adds an OpenACC "enter data" directive to a Schedule.
```

Comment:
enter-data

Reply:
It's "!$acc enter data" so I've used quotes.